README.md

```diff
@@ -24,7 +24,7 @@ subqueries, common table expressions, and more.  Hive's SQL can also be
 extended with user code via user defined functions (UDFs), user defined
 aggregates (UDAFs), and user defined table functions (UDTFs).
 
-Hive users have a choice of 3 runtimes when executing SQL queries.
+Hive users have a choice of 2 runtimes when executing SQL queries.
```

Comment:
Removing the sentence completely would be even better so that we don't have to update the number every time something changes.

MapReduce is deprecate so I am not sure it it is really an option maybe worth mentioning this as well.

With a little effort the commit summary/JIRA title could become more specific and someone could understand what changed (or is going to change) without looking at the diff.

Reply:
> Removing the sentence completely would be even better so that we don't have to update the number every time something changes.

Agreed! Removed this sentence. Thanks for suggestion!

> MapReduce is deprecate so I am not sure it it is really an option maybe worth mentioning this as well.

IMHO, at present, it seems that we never had a concrete plan for deprecating MapReduce, so I think it's better not to mention MR deprecation here. I am afraid of this will lead to some confusion for users.

> With a little effort the commit summary/JIRA title could become more specific and someone could understand what changed (or is going to change) without looking at the diff.

Will change te tilte. Thanks.

Reply:
MR was officially deprecated in Hive 2.0 (see HIVE-12300).

Reply:
Thanks for the info! I added a note  `Note that MapReduce framework has been deprecated since Hive 2, and Apache Tez is recommended.`